update-imported-docs/reference.yml

```diff
@@ -6,10 +6,12 @@ repos:
   branch: master
   generate-command: |
     cd $GOPATH
-    git clone https://github.com/kubernetes/kubernetes.git src/k8s.io/kubernetes
+    # set the branch, ex: v1.17.0 while K8S_RELEASE=1.17
+    # CAUTION: the ${K8S_RELEASE} must be a valid branch from kubernetes repo,
+    # you should NOT specifiy a release branch which is actually NOT released yet,
+    # e.g. K8S_RELEASE=1.18 while kubernetes v1.18 is still in rc stage.
```

Comment:
nit: the branch named `release-1.18` has been there for a long time. What we want here is a tag (e.g. `v1.18.0`) rather than a branch. So what you want to put here could be something like:

```
CAUTION: The script won't work if you set K8S_RELEASE=1.18 before 1.18 is formally released.
The `v${K8S_RELEASE}.0` string must be a valid tag name from the kubernetes repo, which
is only created after the formal release.
```